ui/intermittent-failures/View.jsx

```diff
@@ -134,7 +134,7 @@ const withView = (defaultState) => (WrappedComponent) => {
           const end = ISODate(moment(endday).utc());
           // create entry for each date in range so graph looks nice.
           while (start <= end) {
-            const sdate = moment(start).format('MMM DD');
+            const sdate = moment(start).utc().format('MMM DD');
```

Comment:
This wouldn't fix the issue because `sdate` is the key for the dictionary, not the loop variable which gets increased (`start`).